Source Packages/java/edu/slu/tpen/transfer/JsonLDExporter.java

```diff
@@ -89,7 +89,7 @@ public JsonLDExporter(Project proj, User u) throws SQLException, IOException {
 
          List<Map<String, Object>> pageList = new ArrayList<>();
          for (Folio f : folios) {
-            pageList.add(buildPage(proj.getProjectID(), projName, f, u, man));
+            pageList.add(buildPage(proj.getProjectID(), f, u, man));
```

Comment:
Is this really called for?

Reply:
Hahaha I didn't do it on purpose!  I would have to change things haha.  Let it have an attitude.